src/constants/common.ts

```diff
@@ -382,6 +382,25 @@ export const ALLOWED_ICON_STATUSES = [
   'warning',
 ] as const;
 
+/**
+ * xsm: 12px |
+ * sm: 16px |
+ * rg: 18px |
+ * md: 20px |
+ * lg: 24px |
+ * xl: 28px |
+ * xxl: 30px |
```

Comment:
Why not making this comments in the particular lines?
For example:
```
  xsm: 'xsm', // 12px
```

I think it will be more readable.

Reply:
But it wont show as a comment when you hover over `ICON_SIZES`. That was what I was aiming for

Reply:
So maybe do this in a following way so when you hover the actual property it will show you the value:
```
ICON_SIZES = {
  /** 12px */
  xs: 'xs',
  ...
}
```

Reply:
I wanted the developer to be able to see all the values and select the right value right away, if that doesn't provide any value lets go with what you're suggesting

Reply:
I understand your point of view. But this object does not set the px values. It's only a name map. The CSS actually sets the px size.